platforms/android/library/src/main/java/io/element/android/wysiwyg/utils/ViewLazyViewModelExtension.kt

```diff
@@ -22,6 +32,18 @@ inline fun <reified VM : ViewModel> View.viewModel(
                         ?: modelClass.newInstance()
                 }
             }
-        }
+        },
     )
 }
+
+fun getBaseContext(context: Context): Context {
```

Comment:
Maybe return a `ViewModelStoreOwner` and change to `Context.getViewModelStoreOwner()`

```
    fun Context.getViewModelStoreOwner(): ViewModelStoreOwner {
        var currentContext = this
        while (currentContext !is ViewModelStoreOwner) {
            if (currentContext is ContextWrapper) {
                currentContext = currentContext.baseContext
            } else {
                error("There is no base context that is a ViewModelStoreOwner")
            }
        }
        return currentContext
    }
```